### DISCUSSION TITLE
Motion

### DISCUSSION THREAD

**Discussion by Reflexeye** [gopro/labs#876] [Q&A]
I'm getting Invalid QR code on my Hero 12 - even using the example in the documentation.
...so, I factory reset, updated again and reinstalled labs firmware and it worked!

**Comment by dnewman-gpsw** [comment]
Share the text of the QR code, and the camera firmware version.

**Comment by Reflexeye** [comment]
I factory reset and reinstalled firmware - now works. Many thanks.

On Mon, 22 Apr 2024 at 22:06, David Newman ***@***.***> wrote:

> Share the text of the QR code, and the camera firmware version.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/876#discussioncomment-9193724>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BH2HEBSIXLNMWBUGL4SVPRDY6V3UXAVCNFSM6AAAAABGTNTF5KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCOJTG4ZDI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Jem.
